tools/ml_unittests.sh

```diff
@@ -36,7 +36,7 @@ git clone https://github.com/Qiskit/qiskit-machine-learning.git ${ML_DIR}
 
 echo 'Run unit tests with tox'
 pushd ${ML_DIR}
-tox -e gpu-amd || true
+tox -e gpu || true
```

Comment:
Is it enough to just remove amd or shall we mention nvidia or CUDA somewhere?

Reply:
> Is it enough to just remove amd or shall we mention nvidia or CUDA somewhere?

@dlasecki there should be no need to do anything special for NVIDIA cuda as the Pypi torch already supports it.

The `tox.ini` configuration file in ML, contains two environments:
1. the gpu environment just uses the standard Pypi torch library
2. The gpu-amd environment installs the AMD built version of the torch library: `pip install torch torchvision --extra-index-url https://download.pytorch.org/whl/rocm4.5.2`

So I just switched to use the non AMD one.

Reply:
Ok, thanks for explaining.